PetAdoption/src/main/java/dev/mongoose/models/Scheduler.java

```diff
@@ -1,10 +1,106 @@
 package dev.mongoose.models;
+
 /**
  * Scheduler Class, Required two foreign keys to the pet and person
  * 
- * @author TBD
+ * @author Robert TBD
  *
  */
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="scheduler")
 public class Scheduler {
+@Id
+@GeneratedValue()
+@Column(name="schid",updatable =false)
+private int schId;
+@Column(name="timee")	
+private String time;
+@Column(name="datee")
+private String date;
+@ManyToOne
+@JoinColumn(name="petid")
+private Pet petId;
+@ManyToOne
+@JoinColumn(name="userid")
+private PadUser userId;
```

Comment:
May want to change the object names to just pet and user since they are the objects,